### DISCUSSION TITLE
strategic refinement questions

### DISCUSSION THREAD

**Discussion by nicolevasilevsky** [monarch-initiative/mondo#6768] [General]
Discussion board to ask questions about mass obsoletion.

**Comment by nicolevasilevsky** [comment]
What should we do about terms that will be obsoleted, like MONDO:0016495 - TO_BE_OBSOLETED in [this spreadsheet](https://docs.google.com/spreadsheets/d/1sM0jLlQQAfaO4NzuB588QF8kjWdy3ZsJZy29-TJol1U/edit#gid=1407578343)?

**Reply by nicolevasilevsky** [reply]
~assign them to the top level parent, ie MONDO:0005046. They will need a temporary parent, otherwise the QC build will fail when we create the PR. They will be obsoleted when all the spreadsheets are reviewed and the mass obsoletion is done.~

these terms will be obsoleted in the mass obsoletion pipeline, so they can be ignored.

@katiermullen FYI :eyes:

**Comment by nicolevasilevsky** [comment]
Why are some terms targeted for obsoletion and some are not (when they probably should be)? for example, why is term not in the list to be obsoleted: http://purl.obolibrary.org/obo/MONDO_0017026 ('interstitial lung disease specific to adulthood')? There are other "...interstitial lung disease..." terms that are being obsoleted?

**Reply by nicolevasilevsky** [reply]
Sabrina and I reviewed this list of [potential obsoletion candidates](https://docs.google.com/spreadsheets/d/1e8S9sjE2kmnCOvmqI98CQ7HJoSPh8HyV9CEP4ysI07I/edit#gid=2039272934), and some were marked as 'rescue', which means we'll go back and revisit them later. We'll do another round of review and mass obsoletion after this first round.

See more details here:
https://mondo.readthedocs.io/en/latest/editors-guide/mass-obsoletion/#review-ordo-grouping-classes-for-obsoletion

**Comment by LCCarmody** [comment]
I have a more specific, nuanced question: Term to be obsoleted: common cystic lymphatic malformation which is child of MONDO:0002013->lymphangioma

So:
MONDO:0002013->lymphangioma
>MONDO:0018720 common cystic lymphatic malformation
              Terms to be orphaned:
               >> MONDO:0018717 | mixed cystic lymphatic malformation
                >>MONDO:0019329 | microcystic lymphatic malformation
               >> MONDO:0019328 | macrocystic lymphatic malformation

However, according to NORD explicitly (and other papers, implicitly), "Several different terms were once used to describe conditions now grouped under the umbrella term “lymphatic malformations.” Such terms include cystic hygroma, lymphangioma, cavernous lymphangioma, cystic lymphangioma and lymphangioma circumscriptum. These terms have been abandoned because some of the outdated terms imply a relationship to cancer. Lymphatic malformations are not cancerous and there is no known risk of malignant transformation."

Therefore, there should be a new term 'lymphatic malformation' and it is ONLY child of MONDO:0005833 lymphatic system disorder.

I believe lymphangioma should also be obsoleted.

Not sure how to do that in spreadsheet form...

**Reply by nicolevasilevsky** [reply]
Creating a new ticket to obsolete this term is best. The term lymphangioma is in NCIT, so we can ask them about it as well.

**Reply by nicolevasilevsky** [reply]
Thanks @LCCarmody!

**Reply by LCCarmody** [reply]
It would also be a NTR, but NCIT can comment on that, too

**Reply by nicolevasilevsky** [reply]
ah yes. Feel free to create one ticket for both requests (and label it New term request and Obsolete)

**Comment by LCCarmody** [comment]
Another: What should you do if the orphaned term should also be obsoleted?

Example:
MONDO:0015823 - TO_BE_OBSOLETED->primary immunodeficiency due to a defect in adaptive immunity
Term to be orphaned:
>> MONDO:0010123 | absent thumb-short stature-immunodeficiency syndrome

Text definition is accurate: "An exceedingly rare, autosomal recessive immune disease characterized by thumb aplasia, short stature with skeletal abnormalities, and combined immunodeficiency described in three sibships from two possibly related families. The skeletal abnormalities included unfused olecranon and the immunodeficiency manifested with severe chickenpox and chronic candidiasis. **No new cases have been reported since 1978**."

**Reply by nicolevasilevsky** [reply]
I'm not sure I understand the question. Are you referring to row 39?

**Reply by nicolevasilevsky** [reply]
and you think this term should be obsoleted? (MONDO:0010123 absent thumb-short stature-immunodeficiency syndrome)

Please create an obsoletion ticket and we can revisit it later.

**Reply by LCCarmody** [reply]
Yes, row 39. Assume that's a standard MONDO ticket? I can do that.

**Reply by nicolevasilevsky** [reply]
there are a lot of examples of Orphanet terms like this- historic diseases that may not actually be diseases. I feel like we've discussed this before but I don't remember if we decided we'd try to obsolete them all or do something else. I'll create a new label called historic disease, can you label the new ticket as such and we can discuss these at some point.

**Reply by nicolevasilevsky** [reply]
yep- use the obsolete term template. Thank  you!

**Comment by sabrinatoro** [comment]
# syndromes

How have we been dealing with "syndromes"?
Should syndromes be reclassified as "syndromic diseases", or should they be more specific? 
for example: we have said that "syndromic retinitis pigmentosa" is not a "retinitis pigmentosa", it is a syndrome that have a major phenotype of retinitis pigmentosa. 
What have you been doing?

**Reply by katiermullen** [reply]
This is a great question. I have been classifying endocrine syndromes as syndromic disease if the endocrine component is part of the disease/phenotype, but not the main component. 

Example: MONDO:0019506 obesity-colitis-hypothyroidism-cardiac hypertrophy-developmental delay syndrome, I (with @nicolevasilevsky's help) classified as a "syndromic disease".

MONDO:0016521 muscular pseudohypertrophy-hypothyroidism syndrome is classified as a 'hypothyroidism' because it is a key component of the syndrome.

@sabrinatoro are there better rules to use?

**Reply by nicolevasilevsky** [reply]
it seems like our design pattern specifies that the syndromic form of the disease is the disease with a syndromic presentation:
https://github.com/monarch-initiative/mondo/blob/master/src/patterns/dosdp-patterns/syndromic.yaml

see MONDO:0020148 'syndromic aniridia'

```
aniridia
 and ('has characteristic' some 'has a syndromic presentation')
```

**Reply by nicolevasilevsky** [reply]
a lot of these grouping classes are tagged for obsoletion but not all of them.

here is an example of an OMIMPS that is grouping syndromes:
https://omim.org/phenotypicSeries/PS309510

We should keep this classification, of course.

**Reply by nicolevasilevsky** [reply]
This one didn't come up on our [candidates spreadsheet](https://docs.google.com/spreadsheets/d/1e8S9sjE2kmnCOvmqI98CQ7HJoSPh8HyV9CEP4ysI07I/edit#gid=2039272934):
MONDO:0850009 'syndromic microspherophakia' 

I am realizing now, probably because it doesn't have any children.

**Reply by sabrinatoro** [reply]
Thank you @nicolevasilevsky and @katiermullen 
Just to summarize and convince myself that I understood correctly: 
- a disease would stay in the branch if it is a "syndromic" version of the disease (ie the actual disease, but a syndromic representation). 
- otherwise, it should be in the "syndromic disease" branch (that would be for most of the "syndrome").

**Reply by nicolevasilevsky** [reply]
that is how I interpret it!

**Reply by nicolevasilevsky** [reply]
we have a ticket somewhere to review syndromic diseases at a later time point. I think this is the best approach for now.

**Reply by nicolevasilevsky** [reply]
Discussed with @sabrinatoro and @twhetzel:
For any syndromic disease, we should classify these as syndromic diseases, regardless if the syndrome has features of all the same type, like a cataract-glaucoma should be a syndromic disease, not an eye disease.

We will have a workshop in the new year to discuss this.

cc @katiermullen @LCCarmody @monicacecilia

**Reply by katiermullen** [reply]
I think that I am now confused by what constitutes a syndromic disease. Can we discuss on the curation call tomorrow?

**Reply by nicolevasilevsky** [reply]
yes, we'll discuss tomorrow!

**Reply by LCCarmody** [reply]
I'm not sure I will make it to the call, but I do have an issue with the classification of 'autoinflammatory syndrome'. It should be classified as a syndromic disease (and it is!), but it should NOT be classified as rheumatic disease...or rheumatic disease needs a new definition. Honestly, I think both. I assume I should write a ticket.

**Reply by monicacecilia** [reply]
We agreed to dump all 'leaving' syndromes to their own branch. 
Will reach to the community to corroborate (upvote / downvote) their new location. 
Will reach out to community via Outreach Call to inform. 
Plan to have updated before December release. 
Will schedule workshop for late Jan'24 with community.
